### ISSUE TITLE
Crash when teleporting to another location from the biome.

### ISSUE THREAD

**Issue by ghost** [Ezhh/shadow_land#4]
Most probable way to reproduce; join game, teleport to a random location. 

Log:

```
0x:20:42: ACTION[Server]: singleplayer joins game. List of players: singleplayer
0x:21:03: ERROR[Main]: ServerError: AsyncErr: environment_Step: Runtime error from mod 'shadow_land' in callback environment_Step(): /home/user/.minetest/mods/shadow_land/darkness.lua:12: attempt to perform arithmetic on a nil value
0x:21:03: ERROR[Main]: stack traceback:
0x:21:03: ERROR[Main]: /home/user/.minetest/mods/shadow_land/darkness.lua:12: in function 'func'
0x:21:03: ERROR[Main]: /usr/share/minetest/builtin/common/after.lua:18: in function </usr/share/minetest/builtin/common/after.lua:4>
0x:21:03: ERROR[Main]: /usr/share/minetest/builtin/game/register.lua:412: in function </usr/share/minetest/builtin/game/register.lua:392>
0x:21:03: ERROR[Main]: stack traceback:
0x:21:03: ACTION[Server]: singleplayer leaves game. List of players: 
```

**Comment by Ezhh** [comment]
From some quick testing, this looks to be because the map didn't load quickly enough after teleporting, so light value probably doesn't exist yet,  meaning it gives a nil.

**Comment by Ezhh** [comment]
@Lokait I think this is fixed by 5b89d0c, but please let me know if you find otherwise.
